deploy_staging.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+docker build -t gcr.io/${PROJECT_NAME_STG}/${DOCKER_IMAGE_NAME}:$TRAVIS_COMMIT ./docker
+
+echo $GCLOUD_SERVICE | base64 --decode -i > ${HOME}/gcloud-service-key.json
+cat ${HOME}/gcloud-service-key.json
+gcloud auth activate-service-account --key-file ${HOME}/gcloud-service-key.json
+
+gcloud --quiet config set project $PROJECT_NAME_STG
+gcloud --quiet config set container/cluster $CLUSTER_NAME_STG
+gcloud --quiet config set compute/zone ${CLOUDSDK_COMPUTE_ZONE}
+gcloud --quiet container clusters get-credentials $CLUSTER_NAME_STG
+
+gcloud docker -- push gcr.io/${PROJECT_NAME_STG}/${DOCKER_IMAGE_NAME}
```

Comment:
Why do we need to use google cloud's private repository hosting service ? This is anyway an open source project. So, a private container hosting doesn't make sense and we'll be paying extra for it. (Google cloud platform isn't cheap) 

We can use Docker Hub instead (https://hub.docker.com) like how we do for Open Event. 

https://hub.docker.com/u/eventyay/